### ISSUE TITLE
Orgasmotron is not vibrating

### ISSUE THREAD

**Issue by sebastian-muthwill** [leedave/Leeds-Flipper-Zero-Applications#40]
I have dark flipper installed in version 0.76 and the flipper does not vibrate. The LED changes color on the different modes but no vibration.

**Comment by leedave** [comment]
That is a very old firmware version you‘re using, why not update?

Generally the flippers vibrator only supports on and off. The app just does this in various intervalls. 

Try connecting via ssh (also possible on lab.flipper.net). 
there is a command to switch the vibration on/off, maybe use it to see if your hardware works as it should.

**Comment by sebastian-muthwill** [comment]
Thanks I will check. For DarkFlipper it seems to be the latest version though.

https://github.com/DarkFlippers/unleashed-firmware/releases/tag/unlshd-076

**Comment by leedave** [comment]
ok, so you're using unleashed firmware. Not sure where you got the Orgasmotron from tho, as it is not included in that FW, but there's different ways of getting it. 

First thing to test. 
- Open Settings
- LCD and Notifications
- Check that Vibro is not set to **off**

This will most likely solve your Problem. If that doesn't work, read on. 



**I setup my test device for Unleashed 76** 

- I did a install from [Web Installer](https://lab.flipper.net/apps/orgasmotron) -> **worked fine**
- I compiled the code -> **worked fine**

I put the newly compiled version in this repo for you to try: 
[Leeds Fap File Repo](https://github.com/leedave/Leeds-Flipper-Zero-Fap-Files/tree/main/Unleashed)
-> just copy orgasmotron.fap onto your SD card where the app is (should be in /apps/misc) 

If it still doesn't work, try this
- open [Web Installer CLI](https://lab.flipper.net/cli)
- Type "vibro 1" and hit enter. The Device should vibrate
- Type "vibro 0" to switch vibration off again

if the device doesn't vibrate, then you probably have a hardware problem.
